### DISCUSSION TITLE
Yolo supervision evaluation based on Ground true with Instance ID

### DISCUSSION THREAD

**Discussion by pouya1984** [roboflow/supervision#1094] [Q&A]
Hi,

I was wondering if you can help me, how to evaluate supervision performance for tracking humans in town center. So I have all-ready manually annotated video in coco dataset with included instance Id:s.  My question is how evaluate tracking performance for my paper based on my ground true. Is it possible to do that by matching my ground true instance id with supervision Instance ID:S?

Thanks

**Comment by LinasKo** [comment]
Hi @pouya1984 :wave: 

We have a small Metrics section that includes `MeanAveragePrecision`. I believe this is the method you could use: https://supervision.roboflow.com/metrics/detection/#supervision.metrics.detection.MeanAveragePrecision.from_detections
